Extension/src/ghAPI.ts

```diff
@@ -0,0 +1,152 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
```

Comment:
We should avoid abbreviations in the file name.  Please rename this file `githubApi.ts` or similar.